BeeSwift/TimerViewController.swift

```diff
@@ -125,6 +147,8 @@ class TimerViewController: UIViewController {
             self.timer = nil
             self.timingSince = nil
         }
+        
+        resetIdleTimer()
```

Comment:
The lifecycle of "when we disable the screen" is a little surprising to me. In particular, we enter "do not sleep" mode when you start the timer, but do not leave it when you stop the timer. I'd suggest either enabling it as soon as the user enters the screen, or disabling it when the timer is not running.